### ISSUE TITLE
Integration 'tplink_router' not found.

### ISSUE THREAD

**Issue by SandrRu** [ericpignet/home-assistant-tplink_router#32]
After install - get error
device_tracker.tplink_router Integration 'tplink_router' not found.

Solution - add row in manifest.json
`"version": "0.6",`

like this:
{
    "domain": "tplink_router",
    "name": "TPLink Router device tracker",
    "documentation": "https://github.com/ericpignet/home-assistant-tplink_router",
    "requirements": [],
    "dependencies": [],
	"version": "0.6",
    "codeowners": ["@ericpignet"]
  }

**Comment by OliverParoczai** [comment]
Thanks! I had the same problem with the phonetrack component

**Comment by RobBrinker** [comment]
Also a big thank you! It worked!

**Comment by KoekieWaus** [comment]
got same problem but this is not a fix for me

**Comment by DaveTiff** [comment]
got same problem but this is not a fix for me either

**Comment by Beenay25** [comment]
Yep, same issue. I've added the **_"version": "0.6",_** line but it hasn't fixed the issue.

**Comment by WeterPeter** [comment]
> Yep, same issue. I've added the **_"version": "0.6",_** line but it hasn't fixed the issue.

Same here, adding the version line did not fix it for me.

**Comment by FredFreegroup** [comment]
2022-04-09 15:38:33 ERROR (SyncWorker_2) [homeassistant.loader] The custom integration 'tplink_router' does not have a valid version key (['0.6']) in the manifest file and was blocked from loading.
